pkg/graph/schema.graphql

```diff
@@ -15,7 +15,7 @@ type CurrentUser {
   launchDarkly: LaunchDarklySettings!
 }
 """
-UUIDs are represented using 36 ASCII characters, for example B0511859-ADE6-4A67-8969-16EC280C0E1A
+UUIDs are represented using 36 ASCII characters, for example B05859-ADE6-4A67-8969-16EC280C0E1A
```

Comment:
Why was this changed?

Reply:
huh, I think I just did this on accident. I'll revert it when I make the other changes.